src/index.html

```diff
@@ -7,8 +7,47 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <img
+          src="images/logo.png"
+          alt="Moyo logo"
+          class="img__logo"
+      >
```

Comment:
you need only two spaces fo idnentation
```suggestion
      <img
        src="images/logo.png"
        alt="Moyo logo"
        class="img__logo"
      >
```